tests/pydantic/test_nullability.py

```diff
@@ -68,26 +68,22 @@ def test_empty_lists_are_ignored():
     assert RootModel(field_a="a", list_items=[]).dict() == {
         "field_a": "a",
     }
-    assert RootModel(
-        field_a="a", list_items=[ListItem(), ListItem(), ListItem()]
-    ).dict() == {"field_a": "a"}
+    with pytest.raises(pydantic.ValidationError):
+        RootModel(**{"field_a": "a", "list_items": [{}, {}, {}]})
 
 
 def test_non_empty_list_items_are_serialized():
-    assert RootModel(
-        field_a="a",
-        list_items=[
-            ListItem(list_field_a="list item 1 a"),
-            ListItem(),
-            ListItem(list_field_a="list item 3 a"),
-        ],
-    ).dict() == {
-        "field_a": "a",
-        "list_items": [
-            {"list_field_a": "list item 1 a"},
-            {"list_field_a": "list item 3 a"},
-        ],
-    }
+    with pytest.raises(pydantic.ValidationError):
+        RootModel(
```

Comment:
The same as above - I'd add a check for usage with class instances and rename the test.

Reply:
Agree. I created an extra issue(#42) to address this.